src/application/deployer.go

```diff
@@ -53,5 +54,15 @@ func StartDeploy(deployConfig types.DeployConfig) error {
 	}
 
 	deployConfig.LogChannel <- "Planning deployment..."
-	return terraform.RunPlan(deployConfig, secrets)
+	terraform.RunPlan(deployConfig, secrets)
+	if err != nil {
+		return err
+	}
+
+	if ok := prompt.Confirm("Do you want to continue?"); ok {
```

Comment:
Can this prompt me more descriptive? This is confirming you want toi actually deploy correct? Also why is this prompt here versus at the beginning?